shell/common/rasterizer.cc

```diff
@@ -461,7 +461,8 @@ RasterStatus Rasterizer::DrawToSurface(flutter::LayerTree& layer_tree) {
 
   if (compositor_frame) {
     RasterStatus raster_status = compositor_frame->Raster(layer_tree, false);
-    if (raster_status == RasterStatus::kFailed) {
+    if (raster_status == RasterStatus::kFailed ||
```

Comment:
@iskakaushik I remember we had discussion on this long time ago. We had some issue with not submitting the frame when the status is "resubmit". And we decided to always submit frame (without the actual rendering) for "resubmit", do you remember what was the rational behind it?

Reply:
FWIW, Android was essentially doing the same thing with `should_run_rasterizer_on_platform_thread_`

Reply:
I didn't see where the `[CATransaction commit]` is called when we skip the frame. We need to call it somewhere other than `SubmitFrame` if we don't call `SubmitFrame`.

Reply:
Done.